### PR TITLE
Testing my eden test patches

### PR DESCRIPTION
**Pull request by milan-zededa**
DO NOT MERGE

### CODE REVIEW

**Review by uncleDecart — APPROVED** [review]
🚢

**Review by uncleDecart — APPROVED** [review]
:shipit:

File: .github/workflows/eden.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ jobs:
         arch: [amd64]
         hv: [kvm]
     if: github.event.review.state == 'approved'
-    uses: lf-edge/eden/.github/workflows/test.yml@0.9.7
+    uses: lf-edge/eden/.github/workflows/test.yml@milan-eden-test
```
Comment (uncleDecart) [inline .github/workflows/eden.yml:62]: eden_version should also be changed, because we're technically pulling it

Comment (uncleDecart) [reply .github/workflows/eden.yml:62]: let's try this, but I guess we need real SHA from head of this branch

**Comment by milan-zededa** [comment]
```
Many Requests - Server message: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```
Will try again tomorrow...

**Review by uncleDecart — APPROVED** [review]
🔥

**Comment by milan-zededa** [comment]
Looks good, submitted my patches to the eden repo: https://github.com/lf-edge/eden/pull/986

**Pull request closed without merging**